src/business.ts

```diff
@@ -112,7 +112,7 @@ export class BusinessApi {
     sortBy?: string,
   } = {}): Promise<{
     success: boolean,
-    body?: BusinessList,
+    business?: BusinessList,
```

Comment:
So is this a `business` or `businesses`? Or something else? This is a `BusinessList`, and your naming scheme has it seemingly looking the same as a single `Business`... is that what you, as a user of this, would expect?